sync/config/triggers.yaml

```diff
@@ -26,6 +26,17 @@ archive: https://github.com/tektoncd/triggers/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: release-v0.26.x
+  displayName: v0.26.1
```

Comment:
Should really be `v0.26.x` but I see previous versions in this file also use a specific version number so it's OK for now.

In future we should try to use the `.x` version, that's the purpose of the release branches, so we can push updates to the release branch if needed without requiring additional changes to the website config.

/lgtm

Reply:
Sure. I would do this in future versions.